src/components/listing-create.js

```diff
@@ -218,6 +222,23 @@ class ListingCreate extends Component {
                 <Link to="/">See All Listings</Link>
               </Modal>
             }
+            {this.state.step === this.STEP.ERROR && (
+              <Modal backdrop="static" isOpen={true}>
+                <div className="image-container">
+                  <img src="images/flat_cross_icon.svg" role="presentation" />
+                </div>
+                There was a problem creating this listing.<br />See the console for more details.<br />
+                <a
+                  href="#"
+                  onClick={e => {
+                    e.preventDefault()
```

Comment:
Style/preference question: Do you prefer for this to be handled down here in the render function rather than inside of `resetToPreview`? I would have done as follows but am wondering if there is a reason not to...

```
onClick={this.resetToPreview}
```
```
resetToPreview(e) {
  e.preventDefault()

  this.setState({ step: this.STEP.PREVIEW })
}
```

Reply:
I see now that you followed the convention that was already there. 👍